src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/HearingLocation.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Builder;
+import lombok.Data;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder(toBuilder = true)
+public class HearingLocation {
+    private String locationType;
+    private LocationId locationId;
+
+    public enum LocationId {
+        court,
```

Comment:
Enums should be all caps. Is LocationID going to be used anywhere else?

Reply:
Fixed this. It will be used by hmc hearing, but no where else in common